eth-types/src/geth_types.rs

```diff
@@ -44,6 +44,28 @@ impl Account {
             && self.code.is_empty()
             && self.storage.is_empty()
     }
+
+    /// Generate an account that has either code or balance
+    pub fn mock_code_or_balance(code: Bytecode) -> Self {
+        let is_empty = code.codesize() == 0;
+        Self {
+            address: Address::repeat_byte(0xff),
+            code: code.into(),
+            nonce: U64::from(!is_empty as u64),
+            balance: if is_empty { 0 } else { 0xdeadbeefu64 }.into(),
+            ..Default::default()
+        }
+    }
+
+    /// Generate an account that has 100 ETH
+    pub fn mock_100_ether(code: Bytecode) -> Self {
+        Self {
+            address: Address::repeat_byte(0xfe),
+            balance: Word::from(10).pow(20.into()),
+            code: code.into(),
+            ..Default::default()
+        }
+    }
```

Comment:
I feel these two APIs shouldn't be here. They are here to de-duplicate the test codes. Let me know if you have thoughts on this.

Reply:
The `Account` struct is located at the eth-types crate. But the flat "test-circuits" is in the zkevm-circuits crate.
Adding a flag to `eth-types` would complicate compilation and save only a little compilation time. 
I'm thinking of cleaning up the MockAccount in the mock crate next time and moving these two methods there.